### PR TITLE
first pass at community structure

### PR DESCRIPTION
**Pull request by andrewbattista**
This PR is a first-pass effort at carving out a space on GeoBlacklight.org to display community roles, technical values, and a project roadmap. This starts work gestured at in our community conversations and in #120 

Everything is subject to review!

### CODE REVIEW

**Comment by andrewbattista** [comment]
@karenmajewicz left a question about how to see these changes. One way is to clone the repository, switch to the branch and then serve it locally.  On your Terminal or command line

```
$ git clone https://github.com/geoblacklight/geoblacklight.github.io.git
```

Then navigate to the directory

```
$ cd ~/geoblacklight.github.io
```

Then checkout this branch

```
$ git checkout 120-roadmap
```

Then serve the site locally

```
$ jekyll serve
```
You will be able to view the site as it will look at [http://127.0.0.1:4000/](http://127.0.0.1:4000/)

**Comment by mejackreed** [comment]
Also available here: https://deploy-preview-121--geoblacklight.netlify.app/

File: _projects/openindexmaps/index.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (mejackreed) [inline _projects/openindexmaps/index.markdown:1]: Can/should we separate this change into a separate pull request? Seems like something we could move forward with?

File: about.md [hunk 2]
```diff
@@ -1,11 +1,74 @@
...
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+- [Community Structure](#community-structure)
+- [Technical Values](#technical-values)
+- [Project Roadmap](#project-roadmap)
 
-You can find the source code for the Jekyll new theme at: [github.com/jglovier/jekyll-new](https://github.com/jglovier/jekyll-new)
+GeoBlacklight is a multi-institutional, open source geospatial data discovery software application. Based on the Blacklight Project, GeoBlacklight began in 2014 and has reached its third major release in 2020. GeoBlacklight has been adopted as an interface for geospatial data discovery by over 25 academic libraries and cultural heritage institutions.
 
-You can find the source code for Jekyll at [github.com/jekyll/jekyll](https://github.com/jekyll/jekyll)
+### Community Structure
```
Comment (mejackreed) [inline about.md:13]: Should we break this out into a separate pull request? That it may not hold up moving forward with the about page and other parts that have been more broadly discussed?

Comment (karenmajewicz) [reply about.md:13]: Do you mean we should include the Technical Values list, but not the Community Structure section?  If so, perhaps we can move the current community structure to a blog post that announces our intention to address the topic. The About page could have a named placeholder and a link to the blog post for more information about our progress.

Comment (mejackreed) [reply about.md:13]: Yep, and just the about page in general. My thought here is that most definitely we should define the community structure, but before that has been figured all out, can we ship the about page. I would imagine it might be a sometime before we have all of the details in place about the community structure.
